pyaedt/emit.py

```diff
@@ -45,44 +271,50 @@ class Emit(FieldAnalysisEmit, object):
         Whether to release AEDT on exit. The default is ``False``.
     student_version : bool, optional
         Whether to open the AEDT student version. The default is ``False``.
```

Comment:
```suggestion
        Whether to start the AEDT student version. The default is ``False``.
```
"Open" seems an odd choice.